### DISCUSSION TITLE
Fastest price update

### DISCUSSION THREAD

**Discussion by osatien** [beeb/pancaketrade#71] [Q&A]
What would be the fastest price update?

Asking bsc node each ms? Or is there a form to get last TX against a contract as event?

**Comment by beeb** [comment]
Not sure what you are trying to ask or to do. Can you explain more in detail?

**Comment by osatien** [comment]
i want the price as soon as possible so the 3 secs delay between calls is slow. i think there is a form to have events and be advised when a tx occurs in the pancakeswap contract?

**Comment by beeb** [comment]
You need your own node to subscribe to contract events as far as I know. This is not covered by this script because most people use public rpc's.

**Comment by osatien** [comment]
i am syncing node in this instant .... hope we can do that and trigger faster updates.

with ur code as is... and my node i will change to ipc and download interval checked to 1s (min eval)

**Reply by beeb** [reply]
The repo is open source, feel free to modify it as you please, but i will not put in the time to support this myself as I see little benefit. For most trades, 5 seconds update time is sufficient.
